### PR TITLE
lib/backup: force copying of parts.json

### PR DESCRIPTION
**Pull request by zekker6**
Copying of parts.json is required because `part.key()` comparison can create same key value for files with different contents. This will result in inconsistent backup being created or restored.

See: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5005

### CODE REVIEW

File: lib/backup/actions/copy.go [hunk 1]
```diff
@@ -84,6 +84,7 @@ func runCopy(src common.OriginFS, dst common.RemoteFS, concurrency int) error {
 	}
 
 	partsToCopy := common.PartsDifference(srcParts, dstParts)
+	partsToCopy = append(partsToCopy, common.FilterPartsForForceCopy(srcParts)...)
```
Comment (f41gh7) [inline lib/backup/actions/copy.go:87]: could this function create duplicates? 

 E.g. some parts were added at the previous step to `partsToCopy` and we add it again by filter.

Comment (zekker6) [reply lib/backup/actions/copy.go:87]: Good point, that is possible. I'll update it to only sync those files once

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
Thanks!

**Comment by valyala** [comment]
Please see the follow-up commit 4d01bc6d52e7f581d5e3db7c6a3f295fc6b3868f , which takes into account `parts.json` files inside `indexdb` folder additionally to `data` folder.

See also f93a7b8457bf48660b836711051e316a97c7edc5 , which consistently uses `/` in `Part.Path` on all the supported operating systems.

**Pull request merged**